### PR TITLE
Videoplayer

### PR DESCRIPTION
**Pull request by realswat1**
I implemented the frontend video player for our mux api. Due to the limits of free api ended live streams do not show but once in progress we get a live feed. 
the test attached shows how the main page looks like once you log in. I apologies for my bad wifi . 
https://www.loom.com/share/681674ac0bc44cd18ead1495c689e979?sid=3305186d-997d-4be5-baf9-aaa6242cf800

### CODE REVIEW

File: exposee-ui/src/components/Mainpage/Main.jsx [hunk 1]
```diff
@@ -1,29 +1,78 @@
...
             </>
           ) : (
             <Link to="/login">Login</Link>
           )}
-          </div>
+        </div>
       </header>
+      <div
+        className="video-list"
+        onMouseUp={handleSwipeUp}
```
Comment (alfonsocheng) [inline exposee-ui/src/components/Mainpage/Main.jsx:61]: Is the handle swipe up working already? I didn’t see it in the test plan 

I think you’re missing a flex direction horizontal to make the swipe up make more sense in the test

**Review by alfonsocheng — APPROVED** [review]

File: exposee-ui/src/components/Mainpage/Main.jsx (hunk 1 above)
Comment (realswat1) [reply exposee-ui/src/components/Mainpage/Main.jsx:61]: it makes the cursor change to the hand or grab making the user think it they can drag  , the pan -y is what makes the vertical swipe.

File: exposee-ui/src/components/Mainpage/Main.css [hunk 2]
```diff
@@ -1,30 +1,59 @@
...
+}
+
+video {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 20px 20px 0 0;
+}
+
+p {
```
Comment (alfonsocheng) [inline exposee-ui/src/components/Mainpage/Main.css:52]: it's generally best practice to give your html elements class names to describe what the element is. eg .video-container is great!

Comment (alfonsocheng) [reply exposee-ui/src/components/Mainpage/Main.css:52]: I don't think the "video" and "p" settings actually do anything in this file. If they don't you can just remove them

**Review by alfonsocheng — APPROVED** [review]

**Comment by alfonsocheng** [comment]
really good test plan! keep it up :)

**Pull request merged**